### ISSUE TITLE
✨ Power Automate - Flow to send leads' emails 

### ISSUE THREAD

**Issue by amankumarrr** [SSWConsulting/SSW.Website#1313]
~Recently, we discovered that our old endpoint is still being accessed from legacy pages (e.g., /ContactUs), but it's not triggering email notifications to our Account Managers. To address this, we need to develop a Power Automate Flow that checks our CRM for leads originating from pages with the `.aspx` suffix. If such leads are found, an email notification should be sent to the Account Managers.~
```[tasklist]
### Tasks
- [ ] ~Email Should be sent out to Managers for any lead origniated from legacy pages~
- [ ] ~Make sure there is a line in the email to remind Account Managers to tell the Website team that this is still happening and it shouldn’t happen~
```

We decided to create a new flow that will be triggered when the lead is created in the CRM

**Comment by amankumarrr** [comment]
As per my conversation with @wicksipedia this morning, instead of creating a new flow, we decided to turn on the old email trigger on CRM and remove the email notification from our new Power Automate flow. 

However, after discussion with @zl24622, we are thinking to create a new Power Automate flow that dispatches email on creation of any lead in CRM. 

**Solution 1**
Turn on the CRM trigger and remove it from the Power Automate Flow.

✅Quick Fix
❌Hard to modify the email template later on
❌Current trigger works only for the enquiry from SSW.Website (not from SSW.People/SSW.Rules)

**Solution 2**
✅Easy to maintain 
✅It provides flexibility to update/modify email template 
✅Can send out email notifications from different sources 
❌Development Cost (est 2 days)



@wicksipedia Please review the solution.

**Comment by wicksipedia** [comment]
Based off our earlier conversation, I was under the impression that the old flow sent emails on CRM lead creation.

Solution 2 is ideal

**Comment by wicksipedia** [comment]
> ❌Development Cost

it's better to have an estimate on the 2 solutions - otherwise this statement is meaningless as there is always a developmnet cost

**Comment by amankumarrr** [comment]
@wicksipedia closing this issue as this has been implemented in this issue #1325
